README.md

```diff
@@ -57,6 +57,7 @@ Registered entry points for aiida.calculations:
 The example folder provides scripts to submit calculations in the calculations folder, and tutorials in jupyter notebook format in the tutorials folder.
 
 A quick demo of how to submit a calculation using the provided example files:
+You might need create a [code](https://aiida.readthedocs.io/projects/aiida-core/en/stable/howto/run_codes.html#how-to-create-a-code) for Janus to be recognised as a code by aiida
```

Comment:
If you look at your README, this all merges together, so needs some spacing. I'm also not sure it really makes sense in this order.

Perhaps we want a (very brief, mostly made up of links) AiiDA configuration section in between installation and usagem with `verdi presto` etc. also moved in? That can suggest `verdi presto` as the quickest way to get started (+ link), plus mention setting up the code, but then we should probably also link the full set up for more control.